Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1003,7 +1003,24 @@ private void IBPlaceOrder(Order order, bool needsNewId, string exchange = null)
             }
             else
             {
-                var ibOrder = ConvertOrder(order, contract, ibOrderId);
+                var outsideRth = false;
+
+                if (order.Type == OrderType.Limit ||
+                    order.Type == OrderType.LimitIfTouched ||
+                    order.Type == OrderType.StopMarket ||
+                    order.Type == OrderType.StopLimit)
+                {
+                    var orderProperties = order.Properties as InteractiveBrokersOrderProperties;
+                    if (orderProperties != null)
+                    {
+                        if (orderProperties.OutsideRth)
+                        {
+                            outsideRth = true;
```

Comment:
nit - `outsideRth = orderProperties.OutsideRth`

Reply:
and strictly speaker LEAN does not have contractions; so we should avoid propagating IB's contracted variables names into LEANs classes == `OutsideRegularTradingHours`